Library/UIFeedbackGenerator+Extensions.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+@available(iOS 10.0, *)
+extension UIFeedbackGenerator {
+  public func ksr_successFeedbackGenerator() {
```

Comment:
Might make sense to omit `Generator` in the method naming because it will always be called like `UIFeedbackGenerator().ksr_success()` (which would duplicate both Feedback and Generator)...so how about `ksr_success()` & `ksr_selection()`